frame/staking/src/pallet/mod.rs

```diff
@@ -610,7 +610,7 @@ pub mod pallet {
 		/// The stakers' rewards are getting paid. \[era_index, validator_stash\]
 		PayoutStarted(EraIndex, T::AccountId),
 		/// A validator has updated their commission
-		CommissionChanged(T::AccountId, Perbill),
+		ValidatorPrefsUpdated(T::AccountId, Perbill, bool),
```

Comment:
I think we could just return the single `ValidatorPrefs` type here, and then not worry about constantly changing this as `ValidatorPrefs` updates

Reply:
Updated